### DISCUSSION TITLE
Help! Blanc Canvas: Something doesn't  function anymore with Chrome

### DISCUSSION THREAD

**Discussion by verberg** [cssbattle/forum#238] [Bugs reports]
<img width="893" alt="Bildschirmfoto 2022-03-19 um 17 46 12" src="https://user-images.githubusercontent.com/94834403/159130438-32a582e8-da50-467b-9e7d-8e65b4f38a15.png">
#83 Supernova and #53 Pastel Logo: I can't see my 100% solution. At #53 the little screen is just white, and at #83 I only see the yellow circle. It seams to be a problem with Chrome because I updated on Version 99.0.4844.74. What shall I do? 

And how I see now: I have the problem in every target! I can't play anymore !

**Comment by gunnarbittersmann** [comment]
Same reason as discussed in https://github.com/cssbattle/forum/discussions/231. Rename `--` to `--a` or similar.

**Reply by verberg** [reply]
Thank's - then I have to change the variable -- into --z ? And afterwards, I delete "z"?

**Reply by gunnarbittersmann** [reply]
Yep. And if you want aid and are willing to pay (not me, but the makers of the CSSBattle), here come the [plug-ins](https://github.com/cssbattle/forum/discussions/239) for both directions.
